foo.py

```diff
@@ -1,6 +1,6 @@
-def hello_world():
-    print("Hello, world.")
+def hello_world(who: str):
```

Comment:
`who` seems like it's too narrow. We maybe want to greet things, so `what` is more appropriate